Common/include/option_structure.hpp

```diff
@@ -1489,6 +1489,7 @@ static const MapType<std::string, ENUM_OBJECTIVE> Objective_Map = {
   MakePair("MOMENT_Y", MOMENT_Y_COEFFICIENT)
   MakePair("MOMENT_Z", MOMENT_Z_COEFFICIENT)
   MakePair("EQUIVALENT_AREA", EQUIVALENT_AREA)
+  //MakePair("EQUIV_AREA", EQUIVALENT_AREA)
```

Comment:
Avoid leaving commented-out code, delete what you don't need.

Reply:
Deleted commented out codes.